Extended Backup and Restore on 10.11 build-in system (#1613)
jellyfin/jellyfin.org: docs/general/administration/backup-and-restore.md

Finally, administrative mistakes, normal operation issues (e.g. filling up a disk), or general bit rot can cause corruption or issues that require a backup to be restored. It's always better to have one than not.

> There are two ways of backing up your jellyfin data. One is with its build in Backup/Restore system and the other is manually.

Reviewer: ```suggestion
There are two ways of backing up your Jellyfin data. One is with its built-in Backup/Restore system and the other is manually.
```

> ## Build-in Backup

Reviewer: ```suggestion
## Built-in Backup
```

> Jellyfins build-in backup system is able to create a backup while your system is online and running as opposed to the manual process that **requires** you to stop jellyfin beforehand.

Reviewer: ```suggestion
Jellyfin's built-in backup system is able to create a backup while your system is online and running, as opposed to the manual process that **requires** you to stop Jellyfin beforehand.
```

However in 10.11 we still recommend performing the backup process during a time of low activity and while no scan is currently active.

### Create a Backup

> To take a new Backup, enter the Jellyfin Dashboard, open the "Backups" tab and click on the "Create Backup" button. The popup will now ask you to select what data you want to backup.

Reviewer: ```suggestion
To take a new Backup, enter the Jellyfin Dashboard, open the `Backups` tab and click on the `Create Backup` button. The popup will now ask you to select what data you want to backup.
```

- Database. Always enabled. Will contain all data from the jellyfin.db or used database provider.
- Metadata. The contents of the metadata folder and depending on your selected options may contain metadata images.
- Subtitles. All extracted subtitles including downloaded ones.
- Trickplay. All trickplay data that is stored not alongside media.

> The Backup system will check for at least 4G of free space in the `/config/backup` folder where backups a written to. However this can easily not be enough if you also backup Subtitles and Trickplay so ensure you have enough free space there.

Reviewer: ```suggestion
The Backup system will check for at least 4GB of free space in the `/config/backup` folder where backups a written to. However this can easily not be enough if you also backup Subtitles and Trickplay, so ensure you have enough free space there.
```
Is this path configurable? If so we should mention how to configure it

Author: Added section with listed paths

> After clicking on the "Create" button all data will be written into a new zip archive.

Reviewer: ```suggestion
After clicking on the `Create` button all data will be written into a new zip archive.
```
